views/components/Graphs/BarGraph.jsx

```diff
@@ -29,29 +29,25 @@ const BarGraph = ({ graph }) => {
         y={20}
         labelComponent={<VictoryLabel />}
       />
-      <VictoryAxis
-        label='Site'
-        style={graphStyles.xAxis}
-      />
+      <VictoryAxis label='Site' style={graphStyles.xAxis} />
       <VictoryAxis
         label='Total(%)'
         dependentAxis
         style={graphStyles.yAxis}
+        tickFormat={(tick) => `${tick}%`}
```

Comment:
is this part of another PR? I don't believe these are percentages yet.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/115/commits/181384a418ea7628c74eb97994f5992d4f1e3f71